src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.tsx

```diff
@@ -91,22 +91,25 @@ const secondTierStyles = css`
 interface Props {
     sectionName?: string;
     pillar: Pillar;
+    ajaxUrl: string;
 }
 
-function buildSectionUrl(sectionName?: string) {
+function buildSectionUrl(ajaxUrl: string, sectionName?: string) {
     const sectionsWithoutPopular = ['info', 'global'];
     const hasSection =
         sectionName && !sectionsWithoutPopular.includes(sectionName);
     const endpoint: string = `/most-read${
         hasSection ? `/${sectionName}` : ''
     }.json`;
 
-    return `https://api.nextgen.guardianapps.co.uk${endpoint}?dcr=true`;
+    return `${ajaxUrl}${endpoint}?dcr=true`;
```

Comment:
There's a lib called `[joinUrl](https://github.com/guardian/dotcom-rendering/blob/6003bcc37248219d3b9d0164fe341bd923194587/src/web/components/lib/joinUrl.ts)` which guards against missmatched slashes which would be good to use here

Reply:
Done